Code/Python/inverseArray.py

```diff
@@ -0,0 +1,55 @@
+'''INVERSE OF AN ARRAY
+Problem statement:   If the array elements are swapped with their correseponding
+                     indices,we get the inverse of an array
+NOTE: The elements must be unique and less than the length of array.
+'''
+#inserts the elements in an array of size n
+def inputElements(n):
+    arr=[]
+    for i in range(n):
+        #enter element to insert in array
+         element=int(input())
+         arr.append(element)
+    print("original array:",arr)
+    return arr
+#function to find inverse of array
+def inverseArray(lst):
+    inverted_array=[]
+    for i in range(0,len(lst)):
+        k=lst[i]
+        inverted_array.insert(k,i)
+    return inverted_array
+
+#main code
+# Enter the number of testcases
+t=int(input()) 
+for i in range(0,t):
+    # enter length of array
+    array_length=int(input())
+    arr=inputElements(array_length)
+    arr2=inverseArray(arr)
+    print("Inverse of array :",arr2)
+
+'''
+TestCases :
```

Comment:
Hey, do not accept test cases. Have a look at few merged prs.

Reply:
You directly need to write 
Test Cases
Input :
Output :

Reply:
> Hey, do not accept test cases. Have a look at few merged prs.

@raksha009 please review now, I have made the suggested changes